packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.tsx

```diff
@@ -213,7 +206,7 @@ export class GroupedList extends BaseComponent<IGroupedListProps, IGroupedListSt
 
       group.isCollapsed = !group.isCollapsed;
       this._updateIsSomeGroupExpanded();
-      this.setState({ }, this.forceUpdate);
+      this.setState({}, this.forceUpdate);
```

Comment:
@aditima I believe you added this, do you know why we wouldn't just call forceUpdate?